site/docs/v1/tech/apis/identity-providers/nintendo.adoc

```diff
@@ -0,0 +1,46 @@
+---
+layout: doc
+title: Nintendo Identity Provider APIs
+description: APIs for creating, retrieving, updating and deleting Nintendo identity providers
```

Comment:
We should use similar language to that of Steam, Twitch, or other typed IdPs. Specifically, you cannot have more than one Nintendo IdP, so we should refer to this in a singular fashion.

Reply:
should be fixed